test/wrappedNodeFetch.test.ts

```diff
@@ -0,0 +1,116 @@
+import { wrappedNodeFetch } from '../integrations/octokit/require';
+import { Response } from 'node-fetch';
+import { createExecutionContext, getExecutionContext} from '../src/context';
+describe('wrappedNodeFetch', () => {
+  it('should call fetch function with correct arguments in record mode', async () => {
+    const mockFetch = jest.fn().mockResolvedValueOnce(new Response());
+    const ctx = {
+      mode: 'record',
+      testId: 'testId',
+      mocks: [],
+      deps: [],     
+    };
+    createExecutionContext(ctx)
+    const wrappedFetch = (wrappedNodeFetch(mockFetch) as any).bind({ fetch: mockFetch });
+    const url = 'http://example.com';
+    const options = {
+      method: 'GET',
+    };
+    
+    const response = await wrappedFetch(url, options);
+    const updatedctx= getExecutionContext().context;
+    const mocks=updatedctx.mocks.length;
+    const deps=updatedctx.deps.length;
+    const responseBody = await response.text();
+    const recordedOutput = updatedctx.mocks[0].Spec.Res.Body;
+    expect(mockFetch).toHaveBeenCalledWith(url, options);
+    expect(response).toBeInstanceOf(Response);
+    expect(mocks).toBeGreaterThan(0);
+    expect(deps).toBeGreaterThan(0);
+    expect(response).toHaveProperty('body');
+    expect(responseBody).toEqual(recordedOutput);
+  });
+
+  it('should return mocked response in test mode', async () => {
+    const mockResponse = new Response('mocked response');
+    const mockFetch = jest.fn().mockResolvedValue(mockResponse);
+    const ctx = {
+      mode: 'test',
+      testId: 'testId',
+      mocks: [
+        {
+          Version: 'V1_BETA2',
+          Name: 'testId',
+          Kind: 'HTTP',
```

Comment:
This should be "Http" because we are comparing with [this](https://github.com/keploy/typescript-sdk/blob/10ff4ebc89a3e99fecb4040cf37c6ba274556d50/src/keploy.ts#L27) constant [here](https://github.com/keploy/typescript-sdk/blob/10ff4ebc89a3e99fecb4040cf37c6ba274556d50/integrations/octokit/require.ts#L168).

Reply:
Got it! Thanks for the help.. have made the changes.